src/applications/financial-status-report/components/householdExpenses/InstallmentContractSummary.jsx

```diff
@@ -105,7 +105,7 @@ const InstallmentContractSummary = ({
                   search: `?index=${index}`,
                 }}
                 heading={bill.purpose}
-                key={index + bill.minPaymentAmount + bill.unpaidBalance}
+                key={index} // Use index as the key
                 onDelete={() => onDelete(index)}
                 showDelete
                 body={billBody(bill)}
```

Comment:
Probably didn't need to change this one but I just updated it to match credit card bills

Reply:
I'd lean away from using just the index for the key, mostly bc [the react docs suggest against it](https://react.dev/learn/rendering-lists#rules-of-keys). I know this is a major edge case with the two values, but maybe we use all three?